main.go

```diff
@@ -203,12 +211,7 @@ func validate(in configInput) error {
 func generate(in configInput) ([]byte, error) {
 	c := &config{
 		XdsServers: []server{
-			{
-				ServerUri: in.xdsServerUri,
-				ChannelCreds: []creds{
-					{Type: "google_default"},
-				},
-			},
+			generateServerConfigFromInput(in.xdsServerUri, in),
```

Comment:
I'm wondering if we could make `generateServerConfigFromInput` return a `[]server` with a single entry instead of the caller doing that part. If we ever have to end up supporting more than one entry in the server config, we can deal with that then. What do you think?